examples/speaker_recognition/conf/config.yaml

```diff
@@ -0,0 +1,139 @@
+#model: "QuartzNet15x5"
+sample_rate: &sample_rate 16000
+repeat: &rep 1
+dropout: &drop 0.5
+separable: &separable False
+time_length: 8
+n_filters: &n_filters 512
+
+model:
+  pl: null
+  train_ds:
+    manifest_filepath: '/data/samsungSSD/NVIDIA/repos/NeMo/examples/speaker_recognition/myExps/datasets/voxceleb1/train/train_manifest.json'
```

Comment:
Let's make it shorter: for example, 'voxceleb1/train/train_manifest.json'

Reply:
Done